kinto/views/buckets.py

```diff
@@ -1,7 +1,7 @@
 from kinto.core import resource
 from kinto.core.events import ResourceChanged, ACTIONS
 from pyramid.events import subscriber
-
+from kinto.authorization import RouteFactory
```

Comment:
We thouht about doing that, but it is more the KintoRouteFactory than the BucketRouteFactory.

I think we shouldn't have multiple RouteFactory for a given project, it makes it harder to follow.